Did someone say "loadouts"?

FlattestGuitar opened the pull request:
Are you bored of the crappy scarves?

Do you find space *a bit* too cold for your tastes?

Do you want to change your life in ways that even Flattest couldn't predict?

MORE LOADOUT STUFF

:cl: FlattestGuitar
tweak: adds a lot of stuff to the loadout system
rscadd: adds the sexy security jacket
rscadd: adds the sexy EMT labcoat
rscadd: adds hooded garments
rscadd: adds winter coats
rscadd: adds hoodies
rscadd: adds tracksuits
rscadd: adds new berets
tweak: makes chaplain and cult garments proper hoodies
/:cl:



I've taken the freedom of porting /tg/'s EMT jacket from *god knows which PR* and winter coats from [#6228](https://github.com/tgstation/tgstation/pull/6228), as well as getting a security jacket suit out of my basement.

![](https://i.gyazo.com/47474058b02630fa9100c0e5f0bc9d33.png)

Also added a swanky new CE beret, you'll find it in their locker, not the loadout system.

I've added the following things to the loadout system:
- cheap lighter
- four types of cigarettes
- a box of matches
- a deck of cards
- pet rock
- existential dread
- the classic non-sunglasses secHUD
- four types of ties
- stethoscope
- two types of holodadges
- nine coloured caps
- three fedoras *tips*
- fez
- flat cap
- top hat
- three hardhats
- security beret
- two security caps
- corporate security uniform
- formal security uniform
- two non-formal security uniforms
- bomber jacket
- two trenchcoats
- four military jackets
- Weltschmerz
- leather jacket
- three ponchos
- SEVENTEEN OH GOD pants and shorts
- security jacket (the funky new one)
- paramedic's labcoat (the funky new one)
- cowboy hat (the funky new one)
- dark thoughts (the funky new ones)
- four berets (three of them are funky and new)
- ten winter coats (the funky new ones)
- seven hoodies (the funky new ones)
- four tracksuits (the funky new ones)

Afterthoughts:
The new winter coats have cold protection and in a few cases proper armor. The security and captain have quite a bit of the stuff, but still are worse than all reasonable alternatives.

The security jacket does not have additional pockets and has worse armor values than the armor, but also protects arms. A small difference, but a difference nonetheless.

I PROBABLY FORGOT A LOT OF THINGS, LET ME KNOW WHAT ELSE TO ADD

TheDZD commented:
You don't need the `:` before any of the changelog tags, only after. (For example, `rscadd:`, not `:rscadd:`)

Kevak commented:
Add collaaaars

FlattestGuitar commented:
Cheers, @TheDZD.

sasanek12 commented:
:-1:  No pants or bandanas

Seriously now 100 x :+1: for more stuff to loadout system.

Regen1 commented:
pAI is a no-go, they are rare for a reason, and gives decent enough tech levels

Regen1 commented on code/modules/research/xenoarchaeology/finds/finds.dm:
```diff
@@ -296,9 +296,7 @@
-			var/list/possible_spawns = list(/obj/item/clothing/head/culthood,
-			/obj/item/clothing/head/magus,
-			/obj/item/clothing/head/culthood/alt,
+			var/list/possible_spawns = list(/obj/item/clothing/head/magus,
```
Why was this changed?

FlattestGuitar replied:
The new cult garments have hoods attached to them, not as separate helmets.

If you find cult robes, you automatically have their hoods, just like in real life. 

#makess13realisticagain

LightningLynx commented:
Any chance the regular, non-NV, non-flash protecting SecHUDs can be added?

Also, not sure if they're unavailable for a reason, but there's two security uniforms, under/rank/security2 and under/rank/dispatch that would be nice to have if for no reason other than more variety for Sec garb

tigercat2000 commented:
> Add collaaaars

kinky

Earthdivine commented:
No Collars plz

TheDZD commented:
> No collars plz

I second this.

Fox-McCloud commented on code/modules/client/preference/loadout/loadout_general.dm:
```diff
@@ -1,3 +1,50 @@
+
+/datum/gear/pai
+	display_name = "a pAI card"
+	path = /obj/item/device/paicard
```
This will probably require a bit of discussion.

FlattestGuitar replied:
A pAI is an extremely cheap thing to manufacture, asking RnD is most times going to get you one. 

Its required tech level is Data 2, which can be achieved very easily.

It also gives you Data 2 tech, which will be possible to obtain as soon as you can manufacture it and since it's so cheap to make I doubt it will be somehow abusable.

Taking a pAI with you won't speed up research, nor will it grant you any very expensive resources. People are struggling with finding masters already, this might make it a bit easier for them.

Iamgoofball replied:
good thing I can start with my HARMALARM roundstart

Fox-McCloud replied:
Yeah, this is my essential issue with it...not to mention that R&D mains will take it purely for the research levels =p

Fox-McCloud commented:
Also, please credit TG for the hooded stuff, since that's hwere that came from.

FlattestGuitar commented:
>I've taken the freedom of porting /tg/'s EMT jacket and winter coats

I really have no idea which PR it was and who made it, as far as I can tell it's been there for a long, long time now.

Also, can someone punch Travis so he reruns or does this still have merge conflicts?

TheDZD commented:
Travis doesn't check for merge conflicts, Github reports those by itself. If it says you have merge conflicts, you definitely have merge conflicts.

Spacemanspark commented:
No collars.

FlattestGuitar commented:
Cheers, conflicts fixed, added a normal secHUD, the two uniforms @LightningLynx mentioned and A LACK OF THE COLLAR, because we're not getting the collar for as long as I'm breathing.

Spacemanspark commented:
Oh wait shit please add the hoodies from Polaris.
I don't have time now, but I'll pull up their locations in their codebase, and sprites later.
...Maybe add berets too.
I like berets.

Fox-McCloud commented on code/modules/clothing/suits/armor.dm:
```diff
@@ -8,18 +8,14 @@
 	item_state = "armor"
 	blood_overlay_type = "armor"
-	flags_size = ONESIZEFITSALL
+	flags = ONESIZEFITSALL
```
Stop that.

Fox-McCloud commented on code/modules/clothing/suits/armor.dm:
```diff
@@ -35,7 +31,7 @@
 	item_state = "bulletproof"
 	blood_overlay_type = "armor"
-	flags_size = ONESIZEFITSALL
+	flags = ONESIZEFITSALL
```
I will shank you

Fox-McCloud commented on code/modules/clothing/suits/armor.dm:
```diff
@@ -120,8 +128,6 @@
 	item_state = "jensencoat"
 	flags_inv = 0
-	species_fit = null
-	sprite_sheets = null
```
Now @KasparoVy will shank you too.

Fox-McCloud commented on code/modules/clothing/suits/armor.dm:
```diff
@@ -217,7 +222,7 @@
 	item_state = "armor"
 	blood_overlay_type = "armor"
-	flags_size = ONESIZEFITSALL
+	flags = ONESIZEFITSALL
```
Shanking is going to commence in 5...

Fox-McCloud commented on code/modules/clothing/suits/armor.dm:
```diff
@@ -239,25 +244,22 @@
+		src.icon_state = "reactiveoff"
+		src.item_state = "reactiveoff"
+		src.add_fingerprint(user)
+	return
```
You undid Kasparov's work

Fox-McCloud commented on code/modules/clothing/suits/jobs.dm:
```diff
@@ -52,7 +48,7 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
 	flags_inv = HIDEJUMPSUIT
-	flags_size = ONESIZEFITSALL
+	flags = ONESIZEFITSALL
```
4....

Fox-McCloud commented on code/modules/clothing/suits/armor.dm:
```diff
@@ -88,8 +84,20 @@
+	icon_state = "secjacket_open"
+	item_state = "hos"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	armor = list(melee = 20, bullet = 15, laser = 20, energy = 5, bomb = 15, bio = 0, rad = 0)
```
lower the bullet by 5.

Fox-McCloud commented:
You reverted @KasparoVy and I's work.

We're going to shank you in your face.

tigercat2000 commented:
oh god @FlattestGuitar what horrors did you do

TheDZD commented:
@tigercat2000 Probably fixed merge conflicts by completely overwriting the upstream changes.

FlattestGuitar commented:
oh for the love of god, what did I do

Fox-McCloud commented:
>It's a pretty good way to fix merge conflicts though, no? :eyes:

:hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho::hocho:

FlattestGuitar commented:
Worry not. I shall fix this and then will submit to shanking, as I completely deserve it. :c

FlattestGuitar commented:
I have NO IDEA what you all are talking about. There absolutely has not been a major mistake involving reverting seven years of hard work in the blink of an eye, especially not one of my doing.

Remember kids, none of this would have happened if I had updated my master before starting work on this PR. Also none of this would have happened if I could actually use GitHub, but that's for total nerds.

Also changed the bullet resist. on the sec jacket to 10.

TheDZD commented:
```
code/game/objects/structures/crates_lockers/closets/secure/chaplain.dm:16:error: /obj/item/clothing/suit/hooded/nun: undefined type path

code/game/objects/structures/crates_lockers/closets/secure/chaplain.dm:17:error: /obj/item/clothing/suit/hooded/chaplain_hoodie: undefined type path
```

Fox-McCloud commented on code/modules/client/preference/loadout/loadout_general.dm:
```diff
@@ -1,3 +1,50 @@
+
+/datum/gear/sechud
+	display_name = "a classic security HUD"
+	path = /obj/item/clothing/glasses/hud/security
```
This is weird, when literally every single one of these jobs spawns in with one.

TheDZD replied:
[But they do protect against flashes.](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/clothing/glasses/hud.dm#L76)

Fox-McCloud replied:
What. Okkkk then.

LightningLynx replied:
the same reason so many officers dont wear vests and helmets all the time?

Ar3nn commented:
>rejoices

KingPhilipIII commented:
> Existential Dread

What was that?

KasparoVy commented:
So this doesn't actually revert any of my or Fox' stuff?

How do hoods work with species_fit and sprite_sheets?

Vivalas commented:
Yo add matchboxes and playing cards.

AceMcLazer commented:
Pants, Shorts and short-shorts, Pancho selection sections would be awesome. (cowboy hat :3 ) Those would be great! IIRC Tiger was doing something similar in another PR?

FlattestGuitar commented:
Okayyyyy

- removed the pAI, it's the best way to up your data tech, would be powergamey to keep it
- added a box of matches
- added a deck of cards
- added three ponchos
- added a cowboy hat FROM POLARIS
- added four tracksuits FROM POLARIS
- added seven hoodies FROM POLARIS *but heavily modified*

- fixed the vox sprites for the coats
![](https://i.gyazo.com/2cd1df1ef730098bfe0b42f5b62929a4.png) ![](https://i.gyazo.com/03150a1dc8782b979d1faec49e53a864.png) ![](https://i.gyazo.com/c31a323953bf9359c54a363a9cc32fe6.png)

- still on the fence about the hoods, the current state seems alright to me, considering you won't wear the hood 24/7, but something seems off here. I'd love if anyone wanted to help out with the sprites, there's a metric ton of these to do.
![](https://i.gyazo.com/7c0bafc799eea5e96c3a51f7564bb597.png) ![](https://i.gyazo.com/32350720056484152cd0089cc29bf31a.png) ![](https://i.gyazo.com/245300ac9a16b0c98f9356b4fb6625c6.png)

- cried over how TERRIBLY OH GOD the vox hoodies look
![](https://i.gyazo.com/fc5a0b9e409fc57fe6bc65c6d592d87d.png) ![](https://i.gyazo.com/896897434598b3e828dbfc90eea88c4d.png)

:bird: :gun: 
:boom: :gun: 

I guess I'll have to sprite those for the birds as well...

Spacemanspark commented:
Thanks.

KasparoVy commented:
Doing sprites for the Vox versions of all the jackets right now to help take some of the load off, might take me a bit as I've a lot else to do today but I'll post it when I'm done :+1:

If you want the hoods taken care of too just say

FlattestGuitar commented:
@KasparoVy, if you do the hoods as well I'm gonna give you my firstborn, that would be super great.

The hoodie sprites are pretty much jackets with different colors, so there isn't much to work on there in terms of shapes if you already have the normal jackets done.

I'd really love to do more of this, but at this point I'm puking with spriting so I'll gladly take some time off.

KasparoVy commented:
Alright, I'll have what I've got posted when it's done

KasparoVy commented:
Hey, @FlattestGuitar, I was working and I couldn't help but notice that the sprites for the MIT hoodie and the Black hoodie are the exact same-- both for the hood up and hood down states. That intentional?

FlattestGuitar commented:
Look at the back. The MIT hoodie has the letters in dark red there, while the "normal" black one doesn't. 

At least that's how it's supposed to be.

KasparoVy commented:
Oh, cool. Thanks for pointing that out

FlattestGuitar commented:
Alrighty, I've added pants, shorts and all other two-legged garments I could find in the code.

Also, you can pick as many coats as your soul desires now, they count the points only once.

Fox-McCloud commented on code/modules/clothing/head/misc.dm:
```diff
@@ -161,6 +162,12 @@
+	name = "cowboy hat"
+	icon_state = "cowboyhat"
+	item_state = "fedora"
+	desc = "There's a new sheriff in town."
```
"There's a new sheriff in town. Make sure to pass him some whiskey"

KasparoVy commented:
Just got back and finished all the track jackets. All that's left is to bang out the hood-up winter coats and the hoods themselves and that's it

FlattestGuitar commented:
Let's just comment the Vox species out of the code. Will be easier that way.

:hocho: :bird: 

Also, have I mentioned Drask sprites? :smirk: :hocho: :ice_cream:

KasparoVy commented:
If you don't mind me making a PR either tonight or tomorrow (or whenever after this gets merged) with the vox sprites, hood sprites and the uncommenting, then sure.

I'm nearly finished anyway.

Just the thought of doing all the Drask sprites on top of these Vox ones sends a chill down my spine. I think I'd rather take the deep freeze, to be honest-- but maybe I'll do that in another PR as well.

KasparoVy commented:
[Kasp-JacketAndHoodSprites.zip](https://github.com/ParadiseSS13/Paradise/files/355419/Kasp-JacketAndHoodSprites.zip)

@FlattestGuitar All done! Just have to replace the appropriate files in your branch with the ones in this ZIP and that should be that.

(Replace the suit.dmi and head.dmi files in icons/mob/species/vox with the files from the 'put in icons-mob-species-vox' folder, and replace the suit.dmi and head.dmi files in icons/mob with the files from the 'put in icons-mob' folder)

FlattestGuitar commented:
> Thanks for being so patient.

Come on, man. You did amazing work and I'd be a terrible person if I was bothered by the timeline. I'm adding it to the PR right now.

As per agreement, I'm faxing you my firstborn son right now. Many thanks.

KasparoVy commented on code/modules/clothing/suits/labcoat.dm:
```diff
@@ -56,3 +56,12 @@
+	icon_state = "labcoat_emt_open"
+	item_state = "labcoat_emt_open"
+	species_fit = null
+	sprite_sheets = null
```
Can go ahead and remove this and the line above once the vox sprites get added, it includes a fitted EMT labcoat

KasparoVy commented on code/modules/clothing/suits/hood.dm:
```diff
@@ -0,0 +1,68 @@
+/obj/item/clothing/suit/hooded
+	action_button_name = "Adjust hood"
+	var/obj/item/clothing/head/hood
+	var/hoodtype = /obj/item/clothing/head/winterhood //so the chaplain hoodie or other hoodies can override this
```
Can go ahead and add 
`species_fit = list("Vox")
 sprite_sheets = list(
 	"Vox" = 'icons/mob/species/vox/suit.dmi'
 	)`

KasparoVy commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -364,6 +538,30 @@
+	ignore_suitadjust = 0
+	suit_adjusted = 1
+	action_button_name = "Open/Close Jacket"
+	adjust_flavour = "unzip"
```
Can go ahead and add the Vox species fitting stuff here too once the new sprites are added

KasparoVy commented on code/game/gamemodes/cult/cult_items.dm:
```diff
@@ -43,16 +43,18 @@
+	hoodtype = /obj/item/clothing/head/culthood
 	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade)
 	armor = list(melee = 50, bullet = 30, laser = 50, energy = 20, bomb = 25, bio = 10, rad = 0)
 	flags_inv = HIDEJUMPSUIT
```
Going to need to put `species_fit = null` and `sprite_sheets = null` here 'cause the only Vox sprites I didn't end up doing were the cult stuff, going to lump that into a PR with a bunch of the other jackets I'd missed

KasparoVy commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -282,11 +283,182 @@
+	body_parts_covered = HEAD
+	cold_protection = HEAD
+	flags = NODROP|BLOCKHAIR
+	flags_inv = HIDEEARS
```
Can add the Vox species fitting stuff and remove the species fitting from children here too once once the new sprites are added

FlattestGuitar commented:
Actually, I'd rather not add the sheets globally, but go after specific subtypes of the hooded suit. That will make it less messy for people in the future.

KasparoVy commented:
And not strongarm them into keeping up parity with other species so much haha. Whatever works

FlattestGuitar commented:
:hocho: :hocho: :hocho: :hocho: :hocho: :hocho: 
:bird:      :bird:      :bird:      :bird:      :bird:    :bird:

FlattestGuitar commented:
Okay, I've added four new berets: black, blue, purple and CE's beret. 

Incidentally, the CE's beret can only be found in their locker, not the loadout system.

I think we'll need a future PR to move all the Head stuff to the loadout system so that we don't have cluttered up lockers, but the current UI of the loadout system would make navigating such a densely populated list extremely hard, so that's something for the future.

I'm leaving this PR where it is right now, I'll probably puke the next time I hear the word "sprite".

Consider this ready for review, I suppose.

:heart:

FlattestGuitar commented:
HOLY GODDAMN PIECE OF F--

KasparoVy commented on code/game/gamemodes/cult/cult_items.dm:
```diff
@@ -43,20 +43,22 @@
 	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade)
 	armor = list(melee = 50, bullet = 30, laser = 50, energy = 20, bomb = 25, bio = 10, rad = 0)
 	flags_inv = HIDEJUMPSUIT
-
```
Going to need to put `species_fit = null` and `sprite_sheets = null` here 'cause the only Vox sprites I didn't end up doing were the cult stuff, going to lump that into a PR with a bunch of the other jackets I'd missed

FlattestGuitar replied:
The hooded stuff doesn't globally have the vox sprite sheets. Cult clothes don't inherit them.

Done this for the sake of future spriters that attempt to add more stuff, I'd rather it stayed that way.

taukausanake commented:
but I really wanted collars :(

Spacemanspark commented:
Why though

taukausanake commented:
If I told you I'd have to stab you because you'd want to stab me first

Spacemanspark commented:
Where is Adr when you need him.

Kevak commented:
How about this? Can you provide a legitimate reason _Not_ to include collars as an accessory? Seeing how minimal the tweak would be, and how little it actually affects, is there a legitimate and real reason _Not_ to add them beyond certain people whining about it?

FlattestGuitar commented:
![](https://i.gyazo.com/7cdc827920c752d265b78ce550819140.png)

There we go. I'll add the collars unless someone convinces me not to.

Spacemanspark commented:
Because the only reason people want it is for shitty lewd garbage that we don't need, to be blunt.
Can you give me any other reason besides that in which you'd want to spawn with it?

Regen1 commented:
Here's me convincing you

FUCK. NO.

We're not adding MORE pandering to furries, get that idea straight the fuck out

FlattestGuitar commented:
There are your arguments, @Kevak.

Ready to merge.

Ar3nn commented:
He asked for a legit reason that wasn't whining, and you act like the conflict was resolved with more whining about furries and lewds which are AGAINST SERVER RULES and therefore shouldn't be a factor in a feature implementation or lack thereof.

tigercat2000 commented:
it's legit pandering to furries to add collars you spawn with already on your suit
i would absolutely use it for Woof Larkens, the meme vulvakin :ok_hand:

Spacemanspark commented:
Honestly we should just remove the ability to attach it to a jumpsuit completely.

Kevak commented:
So based on the total lack of providing a reason @Regen1 and @Spacemanspark It appears as though you don't actually have a real mechanical complaint to it. Are you _Capable_ of providing a real reason? Or is it legitimately just whining about things that are agaisnt the server rules anyways and therefore won't come up and if they do the person in question would net themselves a ban? Cause honestly? I'm not seeing any form of real argument against it. Just caustic unpleasentness coming from one person who's supposed to be a community rolemodel IE an admin, and another who's also supposed to be a community rolemodel, IE a mentor. The additude here has much to be desired.

TheDZD commented:
Collars aren't a mechanical change, they're lrgely a fluff one. That said, this is not getting merged as long as they're available.

Regen1 commented:
Because furries legit creep me out, I have no problem with you being a furry, but having the need to walk around in a full blown furry costume and act like you're an animal is the part that's creepy.

You already have **two** races that are fundementally the same, AND the ability to get collars in game, this just makes it so much easier, and also directly panders to furries, which we have way **way** more than enough of. This will just create more drama, and further screw the image of the server up.

FreeStylaLT commented:
I would rather not have collars, but are they even in the PR right now?

Spacemanspark commented:
I gave you a reason, and you disregarded it. That is not my problem.

FlattestGuitar commented:
The collars are currently not in the PR. For the sake of this PR I'd rather another one was made for collars specifically where they can be discussed in-depth.

They are an incredibly minor aspect of this PR and considering the amount of controversy surrounding them I don't think we should be focusing on them at this time.

Kevak commented:
@TheDZD Are you capable of providing a legitimate reason behind such a statement?

@Regen1 That sounds like a significant amount of generalization. Although I am not actually part of the furry community. I can state that the majority of people I am aware of who _Are_ within it do not parade around in nigh terrifying costumes, those things would be uncomfortable as fuck and scare the shit out of anyone. It would be the equivalent of running around in a sports mascot costume. All day every day. Nobody does that if they're in the right mind. I do not believe a single item buried within the loadout selection would actually induce much drama or would affect the server image in the slightest, you appear to be overexaggerating how much people care about minute tiny details.

As for the rest of your points, Many people have stated that the primary reason they play the mentioned races is due to the aesthetic. Due to the nature of SS13 having tiny sprites, a lot of the potential combinations of har/species/so on are eyepain inducing, hence whey they play other racetypes. Your statement of two of said races being mechanically is something I would very much like changed however. I do not like how most of the races don't feel alien. They are literally aliens, they shouldn't play like humans and to hell with balance. SS13 needs to be more lethal anyways. It's been getting way too tame of late. 

@FreeStylaLT Got a reason for that? And I have no idea if it's in currently.

@Spacemanspark No, You did not provide a reason. You provided a caustic complaint. That is not = to a reason. Please provide a legitimate set of reasons like Regens just did.

tigercat2000 commented:
@Kevak Let's not bring this into walls of text. There are 2 seniors admins, 1 head admin, and 2 maintainers against pet collars. That's absolutely all the reason you need.

Kevak commented:
I personally believe authority should actually state their reasoning behind decisions @tigercat2000 . Transparency has proven many times that it's ideal to do such rather then not. Hence the discussion.

tigercat2000 commented:
@Kevak You're in literally no position to demand that they provide their reasoning however. Most of them simply do not want the reputation of "that server where people can spawn with pet collars on".

Kevak commented:
Did I demand? No. I did not. I stated that it's preferable for authority to state their reasoning. Again, we can take this to a private area to continue the discussion if you wish to avoid clogging up the thread.

Spacemanspark commented:
My reasons are pretty much echoing Regens' to a T, if my reason isn't clear enough.

Regen1 commented:
@Kevak it's a matter of first impression

if a new player joins the server and one of the first things they see is that they can put collars on their character that gives a message I don't want this server to give out

Kevak commented:
Makes sense. I accept the reasoning behind that.

taukausanake commented:
I don't want to make this argument any worse than I guess I've already made it, but I'm that kind of person that tends to start fires and watching them burn so I'm going to say this.

Paradise is already the furry station no one wants it to be. I'm not saying to go balls deep with it. I'm saying you already conceived a bastard child and there's no going back. If you're going to use "furry RP station" as an excuse then you're no better off just killing it now. It's too late for an abortion now

Spacemanspark commented:
That's a rather grim thought.
Maybe we should rectify that mentality.

Regen1 commented:
basically what spaceman said

KasparoVy commented:
lmfao, OOC came to the github

funny joke guys

TheDZD commented on code/modules/clothing/suits/hood.dm:
```diff
@@ -0,0 +1,68 @@
+	..()
+
+/obj/item/clothing/suit/hooded/proc/RemoveHood()
+	src.icon_state = "[initial(icon_state)]"
```
Unnecessary `src.`

Fox-McCloud commented:
The flask, cigar, and pipe are a bit of a concern, IMO.

The flask there's normally only a couple of and is essentially a free 60 unit reagent container, the pipe is normally $500 and only available at the cargo computer, and cigars are premium coin-only items from cig machines.

FlattestGuitar commented:
For the sake of keeping premium items premium, removed the cigar, flask and smoking pipe.

Fox-McCloud commented on code/modules/clothing/suits/hood.dm:
```diff
@@ -0,0 +1,68 @@
+
+/obj/item/clothing/suit/hooded/proc/ToggleHood()
+	if(!suit_adjusted)
+		if(ishuman(src.loc))
```
`src`

Fox-McCloud commented on code/modules/clothing/suits/hood.dm:
```diff
@@ -0,0 +1,68 @@
+/obj/item/clothing/suit/hooded/proc/ToggleHood()
+	if(!suit_adjusted)
+		if(ishuman(src.loc))
+			var/mob/living/carbon/human/H = src.loc
```
`src`

Fox-McCloud commented on code/modules/clothing/suits/hood.dm:
```diff
@@ -0,0 +1,68 @@
+				return
+			else if(H.equip_to_slot_if_possible(hood,slot_head,0,0,1))
+				suit_adjusted = 1
+				src.icon_state = "[initial(icon_state)]_hood"
```
`src`

TheDZD commented:
Here's some math for a sec officer being smacked with a toolbox, comparing the armor vest to the armor jacket here (factors in helmet, security jumpsuit armor, and `ran_zone()`). Average damage for the vest will be listed first, followed by average damage for the jacket.
- When attacking the chest or groin, a security officer is somehwat better off with the armor vest (7.04 vs 7.36).
- When attacking a leg, a security officer is marginally better off with the armored jacket (9.04 vs 8.96).
- When attacking a hand or foot, a security officer is marginally better off with the armored jacket (9.84 vs 9.76).
 - Unrelated, but these are the best targets to aim for when fighting a sec officer.
- When attacking an arm, a security officer is significantly better off with the armored jacket (9.04 vs 7.36).
- When attacking the head, a security officer is marginally better off with the armored jacket (7.84 vs 7.76).

I did all this math with a really shitty spreadsheet, although I can clean it up a bit if anybody wants to see it. Didn't do the math with energy or bullet armor due to the fact that ranged weapons' accuracy varies with distance, rather than just using the standard 80% accuracy for `ran_zone()`.

That said, I would tone down the armor values on this a bit more, so that it's more or less in line with the armor vest. Oh, and as you might have noticed, [ran_zone](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/mob_helpers.dm#L97-L117) is a really weird proc, and you're more likely to hit an individual leg, arm, hand, or foot if you miss your original target, than you are to hit the head or the foot (individually).

FlattestGuitar commented:
I honestly couldn't be bothered making another spreadsheet and I don't understand the system anyway.

I've lowered the armor value to 15, that should make it more comparable.

TheDZD commented:
Same issue probably still exists with the laser armor value.
